spec/nelly_spec.rb

```diff
@@ -2,30 +2,32 @@
 require 'spec_helper'
 require 'support/db/active_record'
 
-class Post < ActiveRecord::Base
+class Cacheable < ActiveRecord::Base
   acts_as_cacheable
 end
 
-class Comment < ActiveRecord::Base
+class NonCacheable < ActiveRecord::Base
 end
 
-describe Post do
+describe Cacheable do
   describe '.cache_key' do
     let(:updated_at) { Time.now }
-    let!(:cacheable_object) { Post.create!(updated_at: updated_at) }
+    let!(:cacheable_object) { Cacheable.create!(updated_at: updated_at) }
 
-    subject { Post.cache_key }
+    subject { Cacheable }
```

Comment:
I was thinking the same, but it seems like the subject was Cacheable.new instead of Cacheable.
We want to make sure Cacheable class is tested, not instance of it, so I kept it.